sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (Unreleased)
 
 ### Other Changes
+- This package is no longer being maintained, as the service has been retired. There is no replacement package for this package.
 - This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 are no longer supported.
```

Comment:
We only need 
"This package is no longer being maintained, as the service has been retired. There is no replacement package for this package."

Let's delete the other two.